.github/label-commenter-config.yml

```diff
@@ -0,0 +1,56 @@
+comment:
+ 
```

Comment:
```suggestion
```
Not reqd. an empty line between L1 & L3, pl remove [L2](https://github.com/layer5io/linkerd-service-mesh-workshop/blob/c8a1da20465e15f3e98a2796cff88220e6cf1a10/.github/label-commenter-config.yml#L2)

Reply:
Done